app/components/filter-by.js

```diff
@@ -0,0 +1,29 @@
+  import Component from '@ember/component';
+  import { inject as service } from '@ember/service';
+  import { computed } from '@ember/object';
+
+  export default Component.extend({
+    store: service(),
+
+    allUsers: computed(function () {
+      return this.get('store').findAll('user', { reload: true });
+    }),
+
+    myTeamMembers: computed('allUsers.isFulfilled', function () {
+      return this.allUsers.uniqBy('team');
+    }),
+
+
+
+
+    actions: {
+      isFilter: async function (user) {
+        let getUser = await this.get('store').findAll('user');
```

Comment:
No need to use findAll again here. You can use `this.allUsers`.

Reply:
I have fixed this change.